tests/libs/common/common_tests.cpp

```diff
@@ -40,6 +40,9 @@ ebpf_test_pinned_map_enum()
         goto Exit;
 
     REQUIRE(map_count == pinned_map_count);
+    REQUIRE(map_info != nullptr);
+    if (map_info == nullptr)
```

Comment:
Just a question: is this needed?  i.e., if the REQUIRE in the previous line is false, does Catch2 continue or abort the rest of the test?

Reply:
It aborts, but Code Analysis doesn't detect it. It flags this as a nullptr deref.